### ISSUE TITLE
why i use with Autolayout ,the scrollview contentoff would change!!? 

### ISSUE THREAD

**Issue by zeqinjie** [andreamazz/AMScrollingNavbar#128]
Put five  tableview in scrollview. When I set contentoff offset to the fifth, sliding hidden navigationbar but th scrollview contentoff 0 ,how to  keep the scrollview contentoff is the fifth,how to  solve this problem ,thx

**Comment by andreamazz** [comment]
I'm having an hard time understanding the issue... to put it lightly.  
Can you rephrase, or even better, provide a sample showing the issue?

**Comment by zeqinjie** [comment]
I have been solved，THX,but I met another problem，in your offer DEMO the tableview,when it is collapsed, and  application did enter Background  ,i found the tableview contentOffset is error

**Comment by andreamazz** [comment]
Mmmm yeah, somehow the `didBecomeActive` method got turned around. I really need to add tests to this.

**Comment by andreamazz** [comment]
Closed with version `1.3.8`
